test/test_structured_1d.jl

```diff
@@ -58,6 +58,20 @@ end
     end
 end
 
+@trixi_testset "elixir_advection_float128.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_float128.jl"),
+                        l2=Float128[6.49879312655540217059228636803492411e-09],
+                        linf=Float128[5.35548407857266390181158920649552284e-08])
```

Comment:
Can you please adjust the tolerances to the higher-precision floating point types as well? See, e.g., https://github.com/trixi-framework/Trixi.jl/blob/806c1b9b76586de7cd15a691ed9e7f65f997fe2d/test/test_trixi.jl#L21 - although that may need to be improved... 🙈

Reply:
Any reason to not simply use 

```julia
        atol_default = 500 * eps(RealT)
        rtol_default = sqrt(eps(RealT))
```

For all `RealT`?

Reply:
What we get from the macro is the symbol `:Float64`, not the actual type `Float64`...

Reply:
Ah yeah I see :+1: